keras/src/layers/attention/multi_head_attention.py

```diff
@@ -104,6 +105,7 @@ def __init__(
         use_bias=True,
         output_shape=None,
         attention_axes=None,
+        use_flash_attention=None,
```

Comment:
Please document it in the docstring. Also I think it should just be `flash_attention` to match the op API